docs/plugins/distributing.rst

```diff
@@ -60,3 +60,25 @@ that in the `Plugin Repository's help pages <http://plugins.octoprint.org/help/r
 If you used the `OctoPrint Plugin Cookiecutter Template <https://github.com/OctoPrint/cookiecutter-octoprint-plugin>`_
 when creating your plugin, you can find a prepared registration entry ``.md`` file in the ``extras`` folder of your
 plugin.
+
+Version management after the official plugin repository release
+---------------------------------------------------------------
+
+Once your plugin is available in the official plugin repository, you probably want to create and distribute new versions. 
+For "beta" users you can use the manual file distribution method, or a more elegant release channels (see below). 
+After you finalized a new plugin version, don't forget to actually update the version in the ``setup.py``, and submit a new release on github. 
+
+Submitting a new release is `rather easy <https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/managing-releases-in-a-repository#creating-a-release>`_, 
```

Comment:
Probably, the whole "how to create release" part is unnecessary, and I should just make a link to the `submit a new release on github`, and keep the "After you published" part and so on?

Reply:
Yeah, I'd avoid repeating GitHub's docs and just point people to them - they should also always be accurately reflecting what the user encounters, so less risk of stuff in our docs going stale if we just link to the source.